server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -151,4 +164,33 @@ private DivTag renderAnswer(long programId, AnswerData answerData) {
                 .withClasses(
                     Styles.FLEX_AUTO, Styles.TEXT_RIGHT, Styles.FONT_LIGHT, Styles.TEXT_XS));
   }
+
+  private static SelectTag renderStatusOptionsDropdown(StatusDefinitions statusDefinitions) {
+    SelectTag dropdownTag =
+        select()
+            .withId("select-status")
```

Comment:
If we update the test assertions to operate based on a user-visible element, then I think we don't need to set an ID here.

Reply:
ack, see the other comment about testing the no-statuses case.